source/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter.cc

```diff
@@ -430,6 +432,17 @@ void JsonTranscoderFilter::initPerRouteConfig() {
   per_route_config_ = route_local ? route_local : &config_;
 }
 
+void JsonTranscoderFilter::maybeExpandBufferSize() {
```

Comment:
maybeExpandBufferLimit?

Reply:
Done (but maybeExpandBufferLimits, even, since there's two of them.)